ci/build_python.sh

```diff
@@ -13,14 +13,16 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+rapids-generate-version > ./VERSION
+
 rapids-logger "Begin py build"
 
 CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
 conda config --set path_conflict prevent
 
 sccache --zero-stats
 
-rapids-conda-retry mambabuild \
+RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry mambabuild \
```

Comment:
Why not just:
```suggestion
RAPIDS_PACKAGE_VERSION=$(rapids-generate-version) rapids-conda-retry mambabuild \
```

Reply:
@madsbk once you resolve this thread either way feel free to merge.

Reply:
Missed this thread was blocking. Looks like Vyas and I posted at the same time in fact

Have added this change in a new PR: https://github.com/rapidsai/kvikio/pull/622

We can decide whether or not to include that